visualizer/index.html

```diff
@@ -56,7 +56,7 @@
   section#info #stats {
     float: right;
     width: 155px;
-    height: 119px;
+    height: 135px;
```

Comment:
Needs a better way to fit the info on the screen. My CSS is very rusty but Ideally we'd place the dprof version and whatnot to the left or right of the other info.